src/rvsdg/egglog_optimizer/loop-optimizations.egg

```diff
@@ -1,12 +1,15 @@
-
+(function unroll_count (i64) i64 :unextractable :merge (max old new))
```

Comment:
Do you still need this?

Reply:
yes, setting the unroll max to 2 will make the test not finishing..

Reply:
Okay, fix after today!